documentation/smg/08_setting_up_jenkins_ci.md

```diff
@@ -325,3 +325,49 @@ withCredentials([string(credentialsId: 'TOKEN_ID', variable: 'secret')]) {
   """
 }
 ```
+
+## Established Jenkins pipelines
+
+### PR
+
+#### Herbert
+
+<img src="./images/08_HerbertPR.svg">
+
+#### Horace
+
+<img src="./images/08_HoracePR.svg">
+
+### Nightly
+
+#### Herbert
+
```

Comment:
For this Herbert nightly diagram, a suitable legend could be (in the text) "This diagram shows the process of triggering and running a build of Herbert based on changes in the master branch following one or more pushes of a developer's branch to origin in the last 24 hours".

The text "Nightly Trigger" might more explicitly be stated "Trigger a new build", with the diamond text above it "Has code changed in last 24 hours".